dev-tools/deploy/preflight.py

```
@@ -285,23 +295,311 @@ def is_missing(value: Any) -> bool:
-        if not isinstance(result, dict) or result.get("disposition") not in safe_dispositions:
+        disposition = result.get("disposition") if isinstance(result, dict) else None
+        if disposition not in SAFE_RECOVERY_DISPOSITIONS:
             return (
                 "fail",
                 f"Recovery compatibility baseline {label} has unsafe or missing disposition: {participant}",
             )
     return ("pass", "")
 
 
+def validate_intervening_erasure_coverage_header(
+    value: Any,
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)